### PR TITLE
Update requirements to allow all Django<2.0

### PR DESCRIPTION
**Pull request by naegelyd**
@jdavisp3 @lucaswiman 

The test matrix was also updated to include Django 1.11. Followup to #16.

### CODE REVIEW

File: baya/tests/test_integration.py [hunk 1]
```diff
@@ -242,10 +242,11 @@ def test_index(self):
...
             else:
-                # In django <1.7 B gets a 404 for app_list.
-                # In django >=1.7 B get sa 403, like you'd expect
-                if StrictVersion('1.7.0') <= StrictVersion(
-                        get_django_version()):
+                # In django <1.7 >1.10 B gets a 404 for app_list.
+                # In django >=1.7 <=1.10 B get sa 403, like you'd expect.
+                django_version = StrictVersion(get_django_version())
+                if (django_version >= StrictVersion('1.7.0')
+                        and django_version < StrictVersion('1.11.0')):
```
Comment (naegelyd) [inline baya/tests/test_integration.py:249]: I am not clear on why this behavior changed in Django 1.11 but it seems to have regressed to the behavior we were seeing in Django versions pre-1.7 in that it returns a 404 in Django 1.11

Comment (jdavisp3) [reply baya/tests/test_integration.py:249]: 🤷‍♂️ Oh, Django.

**Review by jdavisp3 — APPROVED** [review]
🚀 Thanks for this @naegelyd !

**Comment by naegelyd** [comment]
Thanks @jdavisp3

**Pull request merged**